Formula/kamel.rb

```diff
@@ -0,0 +1,42 @@
+class Kamel < Formula
+  desc "Apache Camel K CLI"
+  homepage "https://camel.apache.org/"
+
+  url "https://github.com/apache/camel-k.git",
+    :tag      => "0.3.4",
+    :revision => "c47fb2c85e89852f0fd111d1662f57917030ced5"
+  head "https://github.com/apache/camel-k.git"
+
+  depends_on "go" => :build
+  depends_on "openjdk"
+
+  def install
+    # Build from source & Install
```

Comment:
No need to add comments like this, people reading homebrew formula generally know what `make` does

Reply:
comments removed